.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I think we might need to turn this on https://prettier.io/docs/en/options.html#arrow-function-parentheses